core/arch/arm/plat-d02/platform_config.h

```diff
@@ -73,26 +73,28 @@
 #define TZSRAM_BASE		0x50400000
 #define TZSRAM_SIZE		CFG_CORE_TZSRAM_EMUL_SIZE
 
-#define TZDRAM_BASE		0x50460000
-#define TZDRAM_SIZE		(20096 * 1024)
+#define CFG_TZDRAM_BASE		0x50460000
+#define CFG_TZDRAM_SIZE		(20096 * 1024)
 
 #define CFG_TEE_RAM_START	TZSRAM_BASE
 #define CFG_TEE_RAM_PH_SIZE	TZSRAM_SIZE
-#define CFG_TA_RAM_START	ROUNDUP(TZDRAM_BASE, CORE_MMU_DEVICE_SIZE)
-#define CFG_TA_RAM_SIZE		ROUNDDOWN(TZDRAM_SIZE, CORE_MMU_DEVICE_SIZE)
+#define CFG_TA_RAM_START	ROUNDUP(CFG_TZDRAM_BASE, CORE_MMU_DEVICE_SIZE)
+#define CFG_TA_RAM_SIZE		ROUNDDOWN(CFG_TZDRAM_SIZE, CORE_MMU_DEVICE_SIZE)
 
 #define CFG_TEE_RAM_VA_SIZE	(2 * 1024 * 1024)
 
 #else /* CFG_WITH_PAGER */
 
-#define TZDRAM_BASE		0x50400000
-#define TZDRAM_SIZE		(20 * 1024 * 1024)
+#define CFG_TZDRAM_BASE		0x50400000
+#define CFG_TZDRAM_SIZE		(20 * 1024 * 1024)
```

Comment:
move to **conf.mk**?

Reply:
or let each maintainer do the job when she/he wants to...
don't hesitate to discard my comment.